wire-ios/Wire-iOS/Sources/AppDelegate.swift

```diff
@@ -342,11 +346,16 @@ private extension AppDelegate {
     }
 
     private func createAppRootRouter() {
-
+        
         guard let sessionManager = createSessionManager() else {
             fatalError("sessionManager is not created")
         }
-
+        
+        guard mainWindow != nil else {
+            WireLogger.appDelegate.critical("no mainWindow this should not be possible at this point")
+            return
```

Comment:
What about also adding `assertionFailure` so that we will be aware of the issue when debugging?